### PR TITLE
New content

### PR DESCRIPTION
**Pull request by camiloforero**

### CODE REVIEW

**Comment by camiloforero** [comment]
@mikevic I need a review from someone who isn't me for this one

File: domains/engineering/efficient/backend_deployment.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+worker: bundle exec sidekiq -e $RAILS_ENV
+```
+### New Rails server deployment
+1. On the [Cloud 66 Dashboard](https://app.cloud66.com/dashboard), click on *New Application*.
+  1. If you haven't, link your Github account to your Cloud66 acount
+2. Select the git repository you wish to deploy
+3. Select the branch, the environment, and the application name. Conventions to follow are:
+  - `staging` branch will go to the `Staging` environment, `master` branch will go to the `Production` environment
+  - The application name will be in `Title Case` and end with `API` for backend applications. The environment name is not needed.
+4. Click on `Analize Application` and wait for a couple minutes
```
Comment (mikevic) [inline domains/engineering/efficient/backend_deployment.md]: @camiloforero - spelling of analyse is wrong..

Comment (camiloforero) [inline domains/engineering/efficient/backend_deployment.md]:
```suggestion
4. Click on `Analyze Application` and wait for a couple minutes
```

Comment (camiloforero) [inline domains/engineering/efficient/backend_deployment.md]:
```suggestion
4. Click on `Analyze Application` and wait for a couple minutes
```

Comment (camiloforero) [reply domains/engineering/efficient/backend_deployment.md]: @mikevic fixed it. The 'i' was wrong, now it's fixed. The 'z' was American English, and checking the Cloud66 page it's also the spelling they're using, so I'll keep it for accuracy

**Pull request merged**